sale_promotion_rule/views/sale_promotion_rule.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_sale_promotion_rule_tree" model="ir.ui.view">
+        <field name="model">sale.promotion.rule</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="sequence" widget="handle"/>
+                <field name="name"/>
+                <field name="code"/>
+                <field name="discount_amount"/>
+                <field name="date_from"/>
+                <field name="date_to"/>
+                <field name="restriction_amount"/>
+                <field name="minimal_amount"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_sale_promotion_rule_form" model="ir.ui.view">
+        <field name="model">sale.promotion.rule</field>
+        <field name="arch" type="xml">
+            <form string="Promotion Rule">
+                <header>
```

Comment:
remove

Reply:
@nikul-serpentcs IMO it's a good practive to declare the header element into the base view to allow easy extension of this part of the view.